srcpkgs/apparmor/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'apparmor'
 pkgname=apparmor
-version=3.1.2
+version=3.1.3
 revision=1
 build_wrksrc=libraries/libapparmor
 build_style=gnu-configure
 conf_files="/etc/apparmor.d/local/* /etc/apparmor/*"
 make_dirs="/etc/apparmor.d/disable 0755 root root"
-hostmakedepends="bison flex autoconf automake libtool gettext swig python3 which
+hostmakedepends="bison flex automake libtool gettext swig python3 which
```

Comment:
pkg-config is missing according to the configure log which causes all dependency checks to succeed even if something is missing